src/decorators/classes/DChoice.ts

```diff
@@ -23,7 +23,7 @@ export class DChoice<Type = any> extends Decorator {
     super();
   }
 
-  static create<Type = any>(name: string, value: Type) {
+  static create<Type = any>(name: string, value: any) {
```

Comment:
```suggestion
  static create<T = any>(name: string, value: T) {
```
Using `any` is almost never a good idea 😉